packages/magento-cart-email/EmailForm/EmailForm.tsx

```diff
@@ -71,7 +71,8 @@ const EmailFormBase = React.memo<EmailFormProps>((props) => {
             autoComplete: 'email',
             endAdornment: (
               <WaitForQueries waitFor={isEmailAvailable}>
-                {isEmailAvailable.data?.isEmailAvailable && (
+                {(isEmailAvailable.data?.isEmailAvailable ||
```

Comment:
Isn't `isEmailAvailable` always `true` when the guest checkout is disabled? That makes the `isEmailAvailable.data?.isEmailAvailable` redundant